generators/assets/webpack/webpack.go

```diff
@@ -1,10 +1,10 @@
 package webpack
 
 import (
-	"fmt"
 	"os/exec"
 	"path/filepath"
 
+	"github.com/Sirupsen/logrus"
```

Comment:
The repo is now lower case `sirupsen`. Can you please undo these import changes?